src/Symfony/Component/VarDumper/Dumper/ServerDumper.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Dumper;
+
+use Symfony\Component\VarDumper\Cloner\Data;
+
+/**
+ * ServerDumper forwards serialized Data clones to a server.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ServerDumper implements DataDumperInterface
+{
+    private $host;
+    private $wrappedDumper;
+    private $swallows;
+    private $socket;
+    private $contextProviders;
+
+    /**
+     * @param string                   $host
+     * @param DataDumperInterface|null $wrappedDumper    A wrapped instance used whenever we failed contacting the
+     *                                                   server, or if swallows is false
+     * @param bool                     $swallows         Whether to use or not the wrapped dumper instance on normal
+     *                                                   use
+     * @param callable[]               $contextProviders Callables indexed by name returning a context array
+     */
+    public function __construct($host, DataDumperInterface $wrappedDumper = null, $swallows = true, array $contextProviders = array())
```

Comment:
`$host` would then accept an empty string, which would make it read the env var.